azure-pipelines.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           arch: x86
           compiler: msvc2017
           backend: ninja
+          MESON_RSP_THRESHOLD: 0
```

Comment:
Are these doing anything, since RSP is always used on Windows?

Reply:
They're there to explode to smithereens if somebody turns RSP off.

Reply:
> They're there to explode to smithereens if somebody turns RSP off.

You'll have to unpack that a bit more for me to understand.